plugins/PrivacyManager/Controller.php

```diff
@@ -75,7 +75,32 @@ public function gdprOverview()
     {
         Piwik::checkUserHasSomeAdminAccess();
 
-        return $this->renderTemplate('gdprOverview');
+        $purgeDataSettings = PrivacyManager::getPurgeDataSettings();
+
+        $reportRetention = '';
+
+        if ($purgeDataSettings['delete_reports_older_than'] > 12) {
+            $reportRetention .= floor($purgeDataSettings['delete_reports_older_than']/12) . ' ' . Piwik::translate('Intl_PeriodYears') . ' ';
+        }
+        if ($purgeDataSettings['delete_reports_older_than'] % 12 > 0) {
+            $reportRetention .= floor($purgeDataSettings['delete_reports_older_than']%12) . ' ' . Piwik::translate('Intl_PeriodMonths');
+        }
+
+        $rawDataRetention = '';
+
+        if ($purgeDataSettings['delete_reports_older_than'] > 30) {
+            $rawDataRetention .= floor($purgeDataSettings['delete_reports_older_than']/30) . ' ' . Piwik::translate('Intl_PeriodMonths') . ' ';
+        }
+        if ($purgeDataSettings['delete_reports_older_than'] % 30 > 0) {
+            $rawDataRetention .= floor($purgeDataSettings['delete_reports_older_than']%30) . ' ' . Piwik::translate('Intl_PeriodDays');
+        }
```

Comment:
Not sure how beneficial it is, but it would look a bit better to use the singular translation if the year/day/month is `1`. (ie, `1 year 1 month` instead of `1 years 1 months`)